edgedb-client/src/client.rs

```diff
@@ -263,7 +263,7 @@ impl Connection {
         where F: Future<Output = T>
     {
         if self.ping_interval == PingInterval::Unknown {
-            self.init_ping_interval();
+            self.ping_interval = self.init_ping_interval();
```

Comment:
I would also say that the naming is issue here. `init_xxx` reads as something that modifies internal state rather than returns. So `get_ping_interval()` would make the mistake more visible.

Reply:
Exactly! It's renamed to `calc_ping_interval` in the next PR.